shopinvader/services/abstract_sale.py

```diff
@@ -57,6 +57,14 @@ def _convert_one_line(self, line):
             variant = line.product_id._get_invader_variant(
                 self.shopinvader_backend, line.order_id.partner_id.lang
             )
+        if not variant:
+            # we will only land here if the language
+            # still doesn't match a variant, in which case
+            # we will resort to the lang of the backend
+            backend_lang = line.product_id.shopinvader_bind_ids.filtered(lambda x: x.backend_id == self.shopinvader_backend).lang_id.code
```

Comment:
As I showed you this code is broken because you might get more than one binding.
Moreover, if we want to provide a fallback I'd implement this into `_get_invader_variant` by adding a kw `safe_default=False` which will select the _first_ available variant.